data/devices/logitech-g304.device

```diff
@@ -0,0 +1,9 @@
+[Device]
+Name=Logitech Gaming Mouse G304
+DeviceMatch=usb:046d:c53f
```

Comment:
046d:c53f is receiver's ID, which doesn't usually work with libratbag. Not sure why tests don't fail for that. 
Perhaps `lsusb` shows another Logitech device?
If not, try connecting it in wired mode.